packages/html/src/stepper/tests/stepper-horizontal-rtl.html

```diff
@@ -7,7 +7,7 @@
     <link rel="stylesheet" href="/packages/html/assets/styles.css"/>
     <script src="/packages/html/assets/scripts.js"></script>
 </head>
-<body id="app" class="k-body k-bg-gray-100">
```

Comment:
I think that the test background color is grey on purpose, so that we can see that the elements in the stepper well, although some of them have white background color  without border